src/service/notification-service.ts

```diff
@@ -0,0 +1,33 @@
+import { LocalNotifications } from '@capacitor/local-notifications';
+
+export class NotificationService {
+    /**
+     * Request permission from the user to show notifications
+     * It's necessary to use the sendNotification method
+     */
+    authorize = async () => {
+        await LocalNotifications.requestPermissions()
+    }
+
+    /**
+     * Send a local notification to the user after 5 seconds 
+     * @param title - the title of the notification 
+     * @param body - the body of the notification
+     */
+    sendNotification = async (title: string, body: string, schedule: Date = new Date(Date.now())) => {
```

Comment:
what happened to the ID field?

Reply:
If I understand it right, I changed to a random number instead of a fixed 1